study1/week2/sort-the-people.java

```diff
@@ -0,0 +1,27 @@
+// https://leetcode.com/problems/sort-the-people/submissions/
+class Solution {
+    // Time Complexity: O(nlogn)
+    // Space Complexity: O(1)
```

Comment:
We're constructing a hashmap, a new arrayList, and a new nameArr, is the space complexity `O(1)`?